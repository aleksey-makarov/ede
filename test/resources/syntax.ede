{ "var": 12 }
---
{! EDE_SYNTAX pragma=("{*", "*}") inline=("#@", "@#") !}
{* EDE_SYNTAX penis=("#[", "]#")  *}
{! EDE_SYNTAX inline=("{{", "}}") !}

  #@ var @#
    {{ var }}

  {% for nothing in var %}
    {{ var }}
  {% endfor %}
