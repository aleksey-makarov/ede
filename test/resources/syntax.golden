{! EDE_SYNTAX inline=("{{", "}}") !}

  12
    {{ var }}

  {% for nothing in var %}
    {{ var }}
  {% endfor %}
